.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+on: push
```

Comment:
seems it must be explicitely enabled for pull requests:
"pull_request:"
at least that's what I see in hmat

Reply:
yes but we might have both (`pull_request` +` push`)
However I am wondering if there is some jobs do not run concurrently in that case. I can add specific push on `master`